JPS_Ontology/KBTemplates/TBox/OntoSewage.csv

```diff
@@ -0,0 +1,169 @@
+http://www.ontology-of-units-of-measure.org/resource/om-2/hasUnit,Object Property,,,http://www.ontology-of-units-of-measure.org/resource/om-2/Measure,http://www.ontology-of-units-of-measure.org/resource/om-2/Unit,exactly 1,,http://www.ontology-of-units-of-measure.org/resource/om-2/,
+hasInclination,Object Property,,,https://saref.etsi.org/saref4watr/Pipe,Inclination,,A relationship between a pipe and inclination,https://www.theworldavatar.com/kg/ontosewage/,has inclination
+hasLength,Object Property,,,https://saref.etsi.org/saref4watr/Pipe UNION Shaft UNION Flume,http://www.ontology-of-units-of-measure.org/resource/om-2/Length,,A relationship between a pipe or a flume or a shaft and length,https://www.theworldavatar.com/kg/ontosewage/,has length
+hasWallThickness,Object Property,,,https://saref.etsi.org/saref4watr/Pipe,http://www.ontology-of-units-of-measure.org/resource/om-2/Thickness,,A relationship between a pipe and its wall thickness,https://www.theworldavatar.com/kg/ontosewage/,has wall thickness
+hasConnectionProperties,Object Property,,,https://saref.etsi.org/saref4watr/Pipe,ConnectionProperties,,A relationship between a pipe and connection properties,https://www.theworldavatar.com/kg/ontosewage/,has connection properties
+hasEndpointType,Object Property,,,ConnectionProperties,EndpointType,,A relationship between connection properties of a pipe and endpoint of the connection,https://www.theworldavatar.com/kg/ontosewage/,has endpoint type
+hasBranchPipe,Object Property,,,BranchConnection,https://saref.etsi.org/saref4watr/Pipe,,A relationship between a branch connection and a pipe,https://www.theworldavatar.com/kg/ontosewage/,has branch pipe
+hasMainPipe,Object Property,,,BranchConnection,https://saref.etsi.org/saref4watr/Pipe,,A relationship between a branch connection and a pipe,https://www.theworldavatar.com/kg/ontosewage/,has main pipe
+hasOutletLocation,Object Property,,,https://saref.etsi.org/saref4watr/Pipe,https://schema.org/location,,A relationship between a pipe and location (outlet),https://www.theworldavatar.com/kg/ontosewage/,has outlet location
+hasInletLocation,Object Property,,,https://saref.etsi.org/saref4watr/Pipe,https://schema.org/location,,A relationship between a pipe and location (inlet),https://www.theworldavatar.com/kg/ontosewage/,has inlet location
+http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#hasLocation,Object Property,,,BranchConnection UNION https://saref.etsi.org/saref4watr/Manhole,https://schema.org/location,,,http://www.ontologydesignpatterns.org/ont/dul/DUL.owl,
+hasDistanceFromMainPipe,Object Property,,,BranchConnection,http://www.ontology-of-units-of-measure.org/resource/om-2/Distance,,A relationship of distance between a branch connection and main pipe,https://www.theworldavatar.com/kg/ontosewage/,has distance from main pipe

```

Comment:
What is this? If this is intended to specify the distance of a branch connection from the end of the main pipe, this seems wrong! The branch connection in ON the main pipe, not somewhere away FROM the main pipe, only away FROM one end of the pipe

Reply:
`hasDistanceFromMainPipe` is changed to `relativeDistanceOnMainPipe`. Also, its comment is changed.